### PR TITLE
Add changelog link and delete the repeat download page's catalog

### PR DESCRIPTION
**Pull request by angao**
- add changelog-1.10 and changelog-1.11 link
- the download page's catalog is repeat in hugo's markdown rendering engine and hugo can auto generate catalog through the TOC grammar

### CODE REVIEW

**Comment by angao** [comment]
compare [preview](https://deploy-preview-8691--kubernetes-io-master-staging.netlify.com/docs/imported/release/) with [office website](https://kubernetes.io/docs/imported/release/) the catalog

**Comment by Bradamant3** [comment]
/assign @chenopis 

Andrew -- this one looks good to me, and there's a bonus in the catch of the previously unformatted list of previous releases. But there are stray characters at the bottom of the output that appear to be associated with the call to Google Analytics. Can you tell what's going on there? (Might as well clean up the whole file, yes?)

@angao very nice catch on both fronts -- thank you! Just being picky before we merge.

File: content/en/docs/imported/release/notes.md [hunk 1]
```diff
@@ -2042,5 +1943,7 @@ Release notes of older releases can be found in:
 - [CHANGELOG-1.7.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.7.md)
 - [CHANGELOG-1.8.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.8.md)
 - [CHANGELOG-1.9.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md)
+- [CHANGELOG-1.10.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.10.md)
```
Comment (Bradamant3) [inline content/en/docs/imported/release/notes.md]: Please remove this and the following line. 1.10 should be added after 1.11 ships, and 1.11 shouldn't be in the list at all until 1.12 ships.

**Comment by angao** [comment]
@Bradamant3 so why?

File: content/en/docs/imported/release/notes.md (hunk 1 above)
Comment (angao) [reply content/en/docs/imported/release/notes.md]: ok

Comment (angao) [reply content/en/docs/imported/release/notes.md]: I have remove this, could you review again? thx!

**Comment by angao** [comment]
I have remove this, could you review again? thx!

**Comment by Bradamant3** [comment]
Andrew's busy, and the stray characters have been in place for a while. We'll fix later.

/approve

**Pull request merged**